src/main/java/nl/hr/recipefinder/model/entity/Recipe.java

```diff
@@ -0,0 +1,19 @@
+package nl.hr.recipefinder.model.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.ManyToMany;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+public class Recipe extends BaseEntity {
+  private String name;
+  private String desription;
+  private String instructions;
+  @ManyToMany
+  private List<Picture> pictures;
+}
```

Comment:
I believe this should be `@OneToMany` here: 
A `Recipe` has many `Pictures`, but a `Picture` can only have one `Recipe`.